stacker/commands/stacker/base.py

```diff
@@ -141,8 +141,10 @@ def add_arguments(self, parser):
                  "\"namespace\".")
         parser.add_argument(
             "config", type=argparse.FileType(),
+            nargs='?',
```

Comment:
From my limited understanding of argparse, if `type` is `FileType` and the argument is `-` then it uses `sys.stdin`. [From the docs](https://docs.python.org/3/library/argparse.html):

> One of the more common uses of nargs='?' is to allow optional input and output files:

```
>>>
>>> parser = argparse.ArgumentParser()
>>> parser.add_argument('infile', nargs='?', type=argparse.FileType('r'),
...                     default=sys.stdin)
>>> parser.add_argument('outfile', nargs='?', type=argparse.FileType('w'),
...                     default=sys.stdout)
>>> parser.parse_args(['input.txt', 'output.txt'])
Namespace(infile=<_io.TextIOWrapper name='input.txt' encoding='UTF-8'>,
          outfile=<_io.TextIOWrapper name='output.txt' encoding='UTF-8'>)
>>> parser.parse_args([])
Namespace(infile=<_io.TextIOWrapper name='<stdin>' encoding='UTF-8'>,
          outfile=<_io.TextIOWrapper name='<stdout>' encoding='UTF-8'>)
```

> FileType objects understand the pseudo-argument '-' and automatically convert this into sys.stdin for readable FileType objects and sys.stdout for writable FileType objects: